internal/config/mount_config.go

```diff
@@ -24,9 +24,29 @@ type LogConfig struct {
 	FilePath string      `yaml:"file-path"`
 }
 
+type FileCacheConfig struct {
+	Path                    string `yaml:"path"`
+	Size                    int64  `yaml:"size-mb"`
+	TTL                     int64  `yaml:"ttl-sec"`
+	CacheFileForRandomReads bool   `yaml:"cache-file-for-random-reads"`
+}
+
+type MetadataCacheConfig struct {
+	Capacity int64 `yaml:"capacity"`
```

Comment:
this is going to be sizeInMB. Also why are you touching metadata cache configs now.
Focus only on filecache